src/main/java/com/spotinst/sdkjava/model/converters/admin/account/AccountAdminConverter.java

```diff
@@ -0,0 +1,600 @@
+package com.spotinst.sdkjava.model.converters.admin.account;
+
+import com.spotinst.sdkjava.model.api.admin.account.*;
+import com.spotinst.sdkjava.model.bl.admin.account.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AccountAdminConverter {
```

Comment:
Class name should be OrganizationAdminConverter

Reply:
Updated